### PR TITLE
Og field access bybass

### PR DESCRIPTION
**Pull request by emerham**
We needed a way to grant roles a permission that then could bypass all field access permissions, kinda like how user_1 is super user.

### CODE REVIEW

File: og_field_access/og_field_access.module [hunk 1]
```diff
@@ -25,6 +25,10 @@ function og_field_access_field_access($op, $field, $entity_type, $entity, $accou
   if ($entity_type == 'user' && !$id) {
     return;
   }
+  // Check if the user has special permission and content is group content.
+  if (user_access('bypass og field access') && og_is_group_content_type($entity_type, $bundle)) {
```
Comment (alesrebec) [inline og_field_access/og_field_access.module]: You have 
```
if (user_access('administer group')) {
     return TRUE;
   }
```
above. Why would you need another bypass permission?

**Comment by alesrebec** [comment]
Can you specify why would OG need this "bypass" permission if you can use "administer group" permission that is implemented few lines above?

```
function og_field_access_field_access($op, $field, $entity_type, $entity, $account) {
  ...
  if (user_access('administer group')) {
    return TRUE;
  }
  ...

// Check if the user has special permission and content is group content.
  if (user_access('bypass og field access') && og_is_group_content_type($entity_type, $bundle)) {
    return TRUE;
  }
...
```

**Comment by emerham** [comment]
For our use case, we didn't want to put the the user or users in every group as a manager, nor give them mange group permissions because they are only site authors. We have about 200 groups, and they cannot be site admins but needed access to edit all fields that are in their site. It is really a unique use case for this situation, so I have a patch up on d.o to use drush make patching.

If it is too unique for general use I'm fine letting it live on as a patch that gets added into the distro, but figured I'd at least let others have a look see.

**Comment by claudiu-cristea** [comment]
Since https://www.drupal.org/project/og/issues/3467694 was merged into the `8.x-1.x` branch, the development has been moved to https://www.drupal.org/project/og. If this change is still actual, please open an issue at https://www.drupal.org/project/issues/og and provide there a _merge request_ similar with this pull request.

Thank you for contributing to Organic groups module.

**Pull request closed without merging**